config/doctrine-encrypt-module.global.php.dist

```diff
@@ -8,31 +8,27 @@
  * You may want to define a different file for development / production
  */
 return array(
-    'doctrine' => array (
-        'encryption' => array (
-            'orm_default' => array (
+    'doctrine' => array(
+        'encryption' => array(
+            'orm_default' => array(
 
                 //The adapter can be a reference to another service locator key, a Fully qualified class name
                 //or an anonymous function. However, it must implement \Zend\Crypt\BlockCipher or
                 //DoctrineEncrypt\Encryptors\EncryptorInterface
-                'adapter' => function() {
-                    $cipher = \Zend\Crypt\BlockCipher::factory('mcrypt');
-                    $cipher->setKey('Test Key');
-                    return $cipher;
-                },
+                # 'adapter' => \DoctrineEncryptModule\Factory\McryptAdapter::class,
 
                 //The reader will read encryption configuration information off of the entities
                 //You probably don't have to change this
-                'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
-            )
+                # 'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
+            ),
         ),
 
         'eventmanager' => array(
```

Comment:
I think the file should be available, to give users the possibility to change the adapter and the reader. We could improve the documentation about the lines. Take a look into the ZfcUser config example: https://github.com/ZF-Commons/ZfcUser/blob/1.x/config/zfcuser.global.php.dist